specs/gluster_object_detail_views.adoc

```diff
@@ -0,0 +1,198 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of volume and host associated with a gluster cluster discovered by Tendrl
+
+== Problem description
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+*Host List*
+
+The details shown for each host are -
+
+* Status
+* Name
+* Role
+* Alerts
+
+The host list should have functionality for 
+
+* filtering by name, role and status.
+* sorting by name and role.
+
+Host list will be sorted by name by default.
+
+View design will look like:
+
+image::hostList.png[Gluster specific host list view]
+
+*Volume List*
+
+The details shown for each volume are -
+
+* Status
+* Name
+* Type
+* Bricks
```

Comment:
This is just a bricks count or something `10 (8 up, 2 down)` ?
The screen design says value `Server` for brick. Do you mean it would list all the brick names with full path in this list?